doc/database-model.md

```diff
@@ -0,0 +1,61 @@
+
+# Database Entity Relationship Diagram
+
+```mermaid
+erDiagram
+    USER {
+        String _id PK
+        String name
+        String googleID
+        String address
+        ObjectId[] dogs
+    }
+    DOG {
+        String _id PK
+        String name
+        String breed
+        Number age
+        Date birth
+        Number weight
+        Boolean neutered
+        String medication
+        Number medicationPurchaseFrequency
+        String vetCardImageUrl
+        String qrChipImageUrl
+        String[] allergies
+        String[] illness
+        ObjectId[] foods
+        ObjectId[] medical_history
+        ObjectId[] appointments
+        ObjectId[] photos
+    }
+    
+    MEDICAL_HISTORY {
```

Comment:
i think this is better to store inside PET entity, anyway, i am missing a relation with the pet